crt_portal/cts_forms/templates/forms/report.html

```diff
@@ -75,7 +77,7 @@ <h3 class="margin-bottom-1">
       {% endif %}
 
       {% if wizard.form.question_groups %}
-        {# Phasing in question grouping so that we can add names and help text to groups of questions #}
+        {# This is for question groups, so that we can add names and help text to groups of questions #}
```

Comment:
@LindsayYoung I think it would be a good idea to pass all questions through a `QuestionGroup`, even if the group has a single question, so that we can have only one question-HTML-rendering blocks instead of two separate paths for rendering HTML for questions.

Reply:
I think we might need more than one template for these pages. Especially once we have protected class, that page is set up as large buttons.

Reply:
@LindsayYoung 👍👍